apps/opencs/view/render/terrainselection.cpp

```diff
@@ -0,0 +1,114 @@
+#include "terrainselection.hpp"
+
+#include <osg/Group>
+
+#include "../../model/world/cellcoordinates.hpp"
+
+#include "worldspacewidget.hpp"
+
+namespace
+{
+    constexpr int cellSize {ESM::Land::REAL_SIZE};
```

Comment:
Looks like "constexpr" is not supported by Visual Studio 2013.

Reply:
[Already spotted](https://github.com/OpenMW/openmw/pull/1414#issuecomment-323972038) :)
I'll probably just change it to 'const' for now, but I guess VS2013 support might have to be dropped sooner or later, especially if OpenMW ever upgrades to C++14.